tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/IValueProvider.cs

```diff
@@ -0,0 +1,26 @@
+// <copyright file="IValueProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#if !NETFRAMEWORK
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore
+{
+    /// <summary>
+    /// IValueProvider for value provider
+    /// </summary>
+    internal interface IValueProvider
```

Comment:
I think this should be a `[DuckCopy]` struct, as it's not used _directly_ in a method constraint, and doesn't call any methods

Reply:
fixed in [3754fbf](https://github.com/DataDog/dd-trace-dotnet/pull/2495/commits/3754fbfcbb54a8c95ba4d1aadc0ad008dc74848e)